app/javascript/components/applications/application_form/AlternateContactSection.js

```diff
@@ -80,21 +84,26 @@ const AlternateContactSection = ({editValues}) => {
               </div>
               <div className='small-4 columns'>
                 <label>Alternate Contact Type Other</label>
-                <Text field='alternate_contact_type_other' />
+                <Text field='alternate_contact_type_other' maxLength={maxLengthMap['alternate_contact_type_other']} />
```

Comment:
one minor note just on formatting, I'd prefer if we broke these one-line statements into multiple lines to be consistent with the way we format other components

Reply:
Done!